app/assets/stylesheets/partials/_customizations.scss

```diff
@@ -61,16 +61,23 @@
   font-variant: normal;
 }
 
+.checkbox {
```

Comment:
This is more of a question: was the goal to have all checkboxes be normal font weight? I had thought Rachel still wanted most of them bold?

Reply:
Hmm good question. I'll look at figma